Sources/Sentry/SentryRateLimitParser.m

```diff
@@ -0,0 +1,55 @@
+#import <Foundation/Foundation.h>
+#import "SentryRateLimitParser.h"
+#import "SentryCurrentDate.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SentryRateLimitParser ()
+
+@end
+
+@implementation SentryRateLimitParser
+
++(NSDictionary<NSString* , NSDate* > * _Nonnull) parse:(NSString *) header {
+    
+    NSMutableDictionary<NSString* , NSDate*> *rateLimits = [[NSMutableDictionary alloc] init];
+    
+    if ([header length] == 0)  {
+        return rateLimits;
+    }
+    
+    NSString* headerNoWhitespaces = [self removeAllWhitespaces:header];
+    
+    NSArray<NSString *>* quotas = [headerNoWhitespaces componentsSeparatedByString:@","];
+    for (NSString* quota in quotas) {
+        
+        NSArray<NSString *>* parameters = [quota componentsSeparatedByString:@":"];
```

Comment:
```suggestion
        NSArray<NSString *> *parameters = [quota componentsSeparatedByString:@":"];
```

Reply:
Can we validate that we actually have `parameters` because we accessing `[0]` `[1]` afterward and just take for granted the format is like this